api/src/Mail/Account.php

```diff
@@ -467,7 +467,10 @@ public function smtpTransport()
 			// Horde use locale for translation of error messages
 			Api\Preferences::setlocale(LC_MESSAGES);
 
-			if (!empty($GLOBALS['egw_info']['server']['sendmail']))
+			$api_config = Api\Config::read('phpgwapi');
+
+			if (!empty($GLOBALS['egw_info']['server']['sendmail']) ||
+				!empty($api_config['sendmail']))
 			{
```

Comment:
Why do you add that code?

Your $api_config is identical to $GLOBALS['egw_info']['server'].